packages/@stimulus/polyfills/index.js

```diff
@@ -2,5 +2,6 @@ import "core-js/fn/object/assign"
 import "core-js/fn/set"
 import "core-js/fn/map"
 import "core-js/fn/promise"
+import "core-js/fn/array/find"
```

Comment:
We’ve been using Array#find all along. Wonder why the tests pass on master currently? https://github.com/stimulusjs/stimulus/blob/01719aff28e775a09699477897bdb0ba64621664/packages/%40stimulus/core/src/inline_action_observer.ts#L81

Either way, we should add it to the list on https://github.com/stimulusjs/stimulus/blob/master/handbook/06_installing_stimulus.md#browser-support

Reply:
2ad2cec